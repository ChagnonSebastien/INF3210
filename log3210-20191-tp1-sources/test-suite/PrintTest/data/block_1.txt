//simple
{
    a = 0;
}

//imbriquer

{
    a = 1;
    {
        a = 2;
    }
}