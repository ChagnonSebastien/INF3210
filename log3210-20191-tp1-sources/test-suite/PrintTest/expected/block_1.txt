ASTProgram
| ASTBlock
| | ASTBlock
| | | ASTStmt
| | | | ASTAssignStmt
| | | | | ASTExpr
| | ASTBlock
| | | ASTStmt
| | | | ASTAssignStmt
| | | | | ASTExpr
| | | ASTBlock
| | | | ASTStmt
| | | | | ASTAssignStmt
| | | | | | ASTExpr
